### PR TITLE
Visually-hidden h2 for first topic curation

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves https://jira.dev.bbc.co.uk/browse/WSTEAMA-156

**Overall change:**
We want a visually hidden h2 for the first curation.

**Code changes:**

- Visually hidden h2 for first curation; visible h2 for proceeding curations

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/TopicPage/Curation/index.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { arrayOf, oneOf, shape, string, number } from 'prop-types';
 import pathOr from 'ramda/src/pathOr';
+import VisuallyHiddenText from '#psammead/psammead-visually-hidden-text/src';
```
Comment (greenc05) [inline src/app/pages/TopicPage/Curation/index.jsx:4]: Should we be importing from psammead still?

File: src/app/pages/TopicPage/Curation/index.jsx [hunk 2]
```diff
@@ -40,11 +41,15 @@ const Curation = ({
...
-      <Subheading a11yID={createID(title || topStoriesTitle)} href={link}>
-        {title}
-      </Subheading>
+  const isFirstCuration = position === 0;
+  const SubheadingComponent = isFirstCuration ? VisuallyHiddenText : Subheading;
+  const id = createID(title || topStoriesTitle);
+
+  return curationLength > 1 && (title || isFirstCuration) ? (
+    <section aria-labelledby={id} role="region">
+      <SubheadingComponent as="h2" a11yID={id} id={id} href={link}>
```
Comment (greenc05) [inline src/app/pages/TopicPage/Curation/index.jsx]: A href isn't a valid attribue for a h2 FYI this would never contain a link

**Comment by greenc05** [comment]
Please check this with all our supported screen readers

File: src/app/pages/TopicPage/Curation/index.jsx (hunk 1 above)
Comment (rebeccamcginn) [reply src/app/pages/TopicPage/Curation/index.jsx:4]: We merged Psammead into Simorgh. This is import from the psammead dir within Simorgh

File: src/app/pages/TopicPage/Curation/index.jsx (hunk 2 above)
Comment (rebeccamcginn) [reply src/app/pages/TopicPage/Curation/index.jsx]: If it is the first curation, we render the VIsuallyHiddenText component as a h2. If it is second+ curation, we render the Subheading component which can contain a link. It is the Subheading component that accepts href and a11yID as props

Comment (greenc05) [reply src/app/pages/TopicPage/Curation/index.jsx]: Ok, currently if I have looked at the correct storybook link the h2 is output with an empy href?

Comment (rebeccamcginn) [reply src/app/pages/TopicPage/Curation/index.jsx]: Prop name updated

**Review by nicola-d-bbc — APPROVED** [review]

**Review by andrewscfc — APPROVED** [review]

**Review by pvaliani — APPROVED** [review]

File: src/app/pages/TopicPage/Curation/index.jsx [hunk 3]
```diff
@@ -50,7 +50,10 @@ const Curation = ({
       <SubheadingComponent as="h2" a11yID={id} id={id} link={link}>
         {title || topStoriesTitle}
       </SubheadingComponent>
-      <Component promos={promos} headingLevel={headingLevel} />
+      <Component
+        promos={promos}
+        headingLevel={isFirstCuration ? 3 : headingLevel}
```
Comment (greenc05) [inline src/app/pages/TopicPage/Curation/index.jsx:55]: Just checking these will only be h3's if there is more than one curation on the page?

**Pull request merged**